flink-shaded-hadoop/flink-shaded-hadoop2/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,31 @@
+flink-shaded-hadoop2
+Copyright 2014-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.google.guava:guava:11.0.2
+- net.java.dev.jets3t:jets3t:0.9.0
```

Comment:
the indentation is different to the other PRs (which use 2 spaces). It's a minor thing, but also something we can easily be consistent about.